shortcode-ui.php

```diff
@@ -61,6 +61,7 @@ function shortcode_ui_load_textdomain() {
  * @return null
  */
 function shortcode_ui_register_for_shortcode( $shortcode_tag, $args = array() ) {
+	$args = apply_filters( 'shortcode_ui_shortcode_args_' . $shortcode_tag, apply_filters( 'shortcode_ui_shortcode_args', $args, $shortcode_tag ) );
```

Comment:
Can these two filters be applied on separate lines for readability? Also, it would be nice to have PHPdoc comments explaining each filter?